ci/deploy.sh

```diff
@@ -52,8 +52,8 @@ mv /tmp/target/.git ${DEPLOY_BUILD_DIR}
 
 cd ${DEPLOY_BUILD_DIR}
 
-git config user.name "Travis CI"
-git config user.email "travis@travis-ci.com"
+git config user.name "CI Pipeline"
+git config user.email "ci@pipeline.com"
```

Comment:
Does Circle CI have a valid email address? Otherwise we could sign the commits with an a8c email.

Reply:
I was going for general CI pipeline in case we would switch again for some reason:) 

I am happy to change the email. Which one did you had in mind?

Reply:
I think we should use an Automattic, just for consistency?